RELEASE-NOTES.md

```diff
@@ -1,5 +1,14 @@
 This file contains the RELEASE-NOTES of the Semantic Forms Select (a.k.a. SFS) extension.
 
+### 3.0.0
+
+not yet released
+
+* #53 Support "mapping property" / "mapping template" in value field (by Alexander Gesinn), requires changes in PF
```

Comment:
The "requires changes in PF" part is a bit opaque to the users. Something like "Only available when using PageForms version x.y.z or later" is much more understandable. Not abbreviating the extension name is also important, as many users might not guess PF stands for PageForms.